projects/packages/codesniffer/composer.json

```diff
@@ -24,14 +24,16 @@
 	},
 	"scripts": {
 		"phpunit": [
-			"@composer update",
 			"./vendor/phpunit/phpunit/phpunit --colors=always"
 		],
 		"test-coverage": [
 			"@composer update",
 			"phpdbg -d memory_limit=2048M -d max_execution_time=900 -qrr ./vendor/bin/phpunit --coverage-clover \"$COVERAGE_DIR/clover.xml\""
 		],
-		"test-php": "./tests/action-test-php.sh"
+		"test-php": [
+			"@composer update",
+			"./tests/action-test-php.sh"
```

Comment:
For this one, you'll have to put the `composer update` call inside that script, after the test of the PHP version.

https://github.com/Automattic/jetpack/pull/20742/checks?check_run_id=3363657918#step:16:802